conf/messages

```diff
@@ -10,7 +10,7 @@ personal-details.firstname.required=Enter your first name.
 personal-details.lastname=Last name
 personal-details.lastname.required=Enter your last name.
 personal-details.nino=National Insurance number
-personal-details.nino.hint=It''s on your National Insurance card, benefit letter, payslip or P60.
+personal-details.nino.hint=It''s on your National Insurance card, benefit letter, payslip or P60. For example, QQ123456C
 personal-details.nino.hint.example=For example, QQ123456C
```

Comment:
I guess we don't need 'personal-details.nino.hint.example' entry in both messages and messages.cy